### PR TITLE
[Label] Add support for tab stops.

### LINKED ISSUE #4327: \t not worked for string in the Label
**Operating system or device:**
Windows 10

**Issue description** (what happened, and what was expected):
**\t** not worked in the method set_text() for Label object
Example:
get_node("Label").set_text('tab test**\t**wwww')

**Steps to reproduce:**
1. Create the Control node
2. Create the Label node in the Control
3. Create new script for Control node
4. Set script text:

```
extends Control

func _ready():
    get_node("Label").set_text('tab test\twwww')

```

**Link to minimal example project** (optional but very welcome):
See screenshot: http://i.imgur.com/Dw6y6YE.png

### PR DESCRIPTION
**Pull request by bruvzg**
Fixes https://github.com/godotengine/godot/issues/4327

### CODE REVIEW

File: doc/classes/Label.xml [hunk 1]
```diff
@@ -69,6 +69,9 @@
 		<member name="structured_text_bidi_override_options" type="Array" setter="set_structured_text_bidi_override_options" getter="get_structured_text_bidi_override_options" default="[]">
 			Set additional options for BiDi override.
 		</member>
+		<member name="tab_stops" type="PackedFloat32Array" setter="set_tab_stops" getter="get_tab_stops" default="PackedFloat32Array()">
+			Aligns text to the given tab-stops.
```
Comment (Calinou) [inline doc/classes/Label.xml:73]: Could an example be shown using a codeblock? How many values should I add to the array in general?

Comment (bruvzg) [reply doc/classes/Label.xml:73]: One value for the const. width stops, or more for different width spacing (list is repeated). Not sure how to best describe it without an image.

Essentially, it's an equivalent of this list in the text editors:
Screenshot 2023-04-16 at 22 56 18

**Comment by akien-mga** [comment]
Looks good overall, but is this a configurable feature in RichTextLabel too?
I don't see it right away and it seems a bit weird for Label to have a more advanced (and relatively niche) feature than RTL.

**Comment by bruvzg** [comment]
> but is this a configurable feature in RichTextLabel too?

I was sure it is, but apparently it's not, RTL is using hard-coded tab size, I will do RTL PR as well.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Needs rebase.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**